pkgs/servers/nextcloud/default.nix

```diff
@@ -28,6 +28,26 @@ let
       inherit packages;
     };
 
+    outputs = [ "out" "compressed" ];
+
+    nativeBuildInputs = [ zopfli brotli ];
+
+    buildPhase = ''
```

Comment:
Please move this into passthru. Someone not wanting compressed data should be required to run this for potentially minutes.